Makefile

```diff
@@ -36,7 +36,7 @@ endif
 # run checks on all files for the repo
 quality:
 	@echo "Running copyright checks";
-	python utils/copyright.py quality $(PYCHECKGLOBS) $(JSCHECKGLOBS) $(MDCHECKGLOBS) $(MDCHECKFILES)
+	python3 utils/copyright.py quality $(PYCHECKGLOBS) $(JSCHECKGLOBS) $(MDCHECKGLOBS) $(MDCHECKFILES)
```

Comment:
If you're running on ubuntu, I might argue for installing `python-is-python3` instead
```
apt-cache search python-is-python3
python-is-python3 - symlinks /usr/bin/python to python
```

Reply:
Oops, didn't meant to commit this. Perhaps this should be a different PR